source/install/install-rhel-66-mysql.rst

```diff
@@ -0,0 +1,66 @@
+..  _install-rhel-66-mysql:
+
+Installing MySQL Database Server
+================================
+
+Install and set up the database for use by the Mattermost server. You can install either MySQL or PostgreSQL. To install PostgreSQL, see :ref:`install-rhel-66-postgresql`.
```

Comment:
Hmm okay, that's a good point - if there isn't a way to properly link to the PostgreSQL database let's remove the link for now then and just say "To install PostgreSQL, see Installing PostgreSQL Database Server below" or something similar.